.docker/.env

```diff
@@ -0,0 +1,4 @@
+﻿MS_SQL_DB_PORT=1433
+SA_PASSWORD=Sashacool1
+DB_USER=sa
+DB_NAME=StreetcodeDb
```

Comment:
You should create at least 2 databases(Streetcode and Streetcode_IntegrationTests)
My suggestion is to create nuke task CreateDatabase that creates all needed databases with simple sql, in that case you can use SA creds in each connection string

Reply:
We are leaving it for the future scaling.